src/main.cpp

```diff
@@ -125,7 +125,7 @@ int main(int argc, char** argv)
     // start minimized if configured
     bool minimizeOnStartup = config()->get("GUI/MinimizeOnStartup").toBool();
     bool minimizeToTray    = config()->get("GUI/MinimizeToTray").toBool();
-    if (minimizeOnStartup) {
+    if (minimizeOnStartup && !minimizeToTray) {
```

Comment:
If this particular issue only affects Linux, the extra condition shouldn't be enforced on other OSes.

Also, the "minimize on startup" checkbox should be disabled when "minimize to tray" is active at the settings window on Linux.

Reply:
Why? I want KeePassXC to start minimized in the tray.

Reply:
@weslly It shouldn't hurt if the condition is enforced in other platforms, though I will add an #ifdef for it if it makes the reason clearer. Also a comment referencing it, now that I think of it.

I concur with @phoerious, "minimize on startup" + "minimize to tray" should be a valid combination, it's specially useful if you have set KeePassXC to auto unlock on boot (and want it to be on the tray upon boot).

Reply:
@phoerious Whoops, I just noticed it doesn't matter since the window is only shown on the next `if`